components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -3508,6 +3508,163 @@ func TestListRules(t *testing.T) {
 	}
 }
 
+func TestListRulesForProject(t *testing.T) {
+	store, db, _ := setup(t)
+	defer db.close(t)
+	defer store.Close()
+
+	cases := map[string]func(*testing.T){
+		"when no rules or projects exist, returns an empty list": func(t *testing.T) {
+			ctx := context.Background()
```

Comment:
[nit] let's move this out of the `cases` definition; there's nothing to be gained from repeating it in every case.

Reply:
Ah, looks we'd better use a fresh one for some cases below to insert projects. That's fine then.